server.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+require('dotenv').load();
+const express = require('express');
```

Comment:
When I run your tests, I get an error that says ``cannot find module express``. You need to ``npm i - S express`` or else nothing will work.